e2e/testcases/stress_test.go

```diff
@@ -224,9 +228,34 @@ func TestStressLargeRequest(t *testing.T) {
 		nt.T.Fatal(err)
 	}
 
-	rootSyncFilePath := "../testdata/root-sync-crontab-crs.yaml"
-	nt.T.Logf("Apply the RootSync object defined in %s", rootSyncFilePath)
-	nt.MustKubectl("apply", "-f", rootSyncFilePath)
+	rootSync := fake.RootSyncObjectV1Beta1(configsync.RootSyncName)
+	reconcilerOverride := v1beta1.ContainerResourcesSpec{
+		ContainerName: reconcilermanager.Reconciler,
+		MemoryLimit:   resource.MustParse("1500Mi"),
+	}
+	if *e2e.GKEAutopilot {
+		reconcilerOverride.MemoryRequest = resource.MustParse("1500Mi")
+	}
+	rootSync.Spec = v1beta1.RootSyncSpec{
+		SourceFormat: string(filesystem.SourceFormatUnstructured),
+		Git: &v1beta1.Git{
+			Repo:      "https://github.com/config-sync-examples/crontab-crs",
+			Branch:    "main",
```

Comment:
is main default now with the latest git-sync? @nan-yu

Reply:
The short answer is no.

`git-sync` v4 uses a different flag `--ref` to specify a branch, a tag or a commit hash. It also consolidates the legacy `--branch` and `--rev` flags: https://github.com/kubernetes/git-sync/blob/v4.2.1/main.go#L358.

If none of these flags are set, it fetches the HEAD of the default remote branch, which may or may not be `main`.

But in Config Sync, if none of these flags are set, we force to use `master` as the default branch of backward compatibility.